### PR TITLE
Improve logging

### PR DESCRIPTION
**Pull request by dblenkus**

### CODE REVIEW

File: resolwe/flow/management/commands/list_docker_images.py [hunk 1]
```diff
@@ -84,9 +93,15 @@ def handle(self, *args, **options):
                         raise CommandError(errmsg)
                     else:
                         # Print error, but keep going
-                        self.stderr.write(errmsg)
+                        pass
+                        logger.error(errmsg)
```
[the remark this answers is not in the document]
Comment (dblenkus) [reply resolwe/flow/management/commands/list_docker_images.py]: Management command are used interactively, so it is nice to print something to stdout/stderr (especially if verbosity is `1`) as there in mo guarantee that logging is configured properly.

File: tests/settings.py [hunk 2]
```diff
@@ -194,3 +194,47 @@
...
+        'console': {
+            'class': 'logging.StreamHandler',
+            'level': 'WARNING',
+            'formatter': 'standard',
+        },
+        'file': {
+            'class': 'logging.handlers.RotatingFileHandler',
+            'filename': debug_file_path,
+            'formatter': 'standard',
+            'maxBytes': 1024 * 1024 * 10,  # 10 MB
```
[the remark this answers is not in the document]
Comment (dblenkus) [reply tests/settings.py]: Yes, it defaults to `0`, so no old logs will be stored.

File: resolwe/flow/managers/listener.py [hunk 3]
```diff
@@ -171,7 +171,7 @@ def handle_update(self, obj, internal_call=False):
         changeset = obj[ExecutorProtocol.UPDATE_CHANGESET]
         if not internal_call:
             logger.debug(
-                __("Resolwe listener: handle_update[{}].", data_id),
+                __("Handling update for data id {}.", data_id),
```
Comment (tjanez) [inline resolwe/flow/managers/listener.py]: I forgot to comment here previously. I would prefer to use:
```
__("Handling update for Data with id {} (handle_update).", data_id),
```

File: resolwe/flow/managers/dispatcher.py [hunk 4]
```diff
@@ -324,6 +342,8 @@ def _prepare_data_dir(self, data_id):
         :return: The prepared data directory path.
         :rtype: str
         """
+        logger.debug(__("Preparing data directory for data id {}.", data_id))
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: For consistency with listener's logging messages, use `logger.debug(__("Preparing data directory for Data with id {}.", data_id))` here.

File: resolwe/flow/managers/dispatcher.py [hunk 5]
```diff
@@ -416,6 +436,8 @@ def _prepare_executor(self, data_id, executor):
             be deployed.
         :rtype: (str, str)
         """
+        logger.debug(__("Preparing executor for data id {}", data_id))
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: Similar as above, I would use `logger.debug(__("Preparing executor for Data with id {}", data_id))` here.

File: resolwe/flow/managers/dispatcher.py [hunk 6]
```diff
@@ -582,11 +600,9 @@ def communicate(self, data_id=None, run_sync=False, verbosity=1, save_settings=T
 
         executor = getattr(settings, 'FLOW_EXECUTOR', {}).get('NAME', 'resolwe.flow.executors.local')
         logger.debug(__(
-            "Manager '{}.{}' sending communicate command on '{}' with executor set to '{}'.",
-            self.__class__.__module__,
-            self.__class__.__name__,
+            "Manager sending communicate command on '{}' triggered by data id {}.",
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: Similar as above, I would use `"Manager sending communicate command on '{}' triggered by Data with id {}.",` here.

File: resolwe/flow/managers/dispatcher.py [hunk 7]
```diff
@@ -644,12 +656,7 @@ def _data_execute(self, data, program, executor, verbosity):
         if not program:
             return
 
-        logger.debug(__(
-            "Manager '{}.{}' preparing Data object {} for processing.",
-            self.__class__.__module__,
-            self.__class__.__name__,
-            data.id
-        ))
+        logger.debug(__("Manager preparing Data object {} for processing.", data.id))
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: For consistency, I would change this to: `logger.debug(__("Manager preparing Data with id {} for processing.", data.id))`.

File: resolwe/flow/managers/dispatcher.py [hunk 8]
```diff
@@ -755,13 +762,7 @@ def process_data_object(data):
...
             )
 
-        logger.debug(__(
-            "Manager '{}.{}' processing communicate command on '{}' for executor '{}'.",
-            self.__class__.__module__,
-            self.__class__.__name__,
-            state.MANAGER_CONTROL_CHANNEL,
-            executor
-        ))
+        logger.debug(__("Manager processing communicate command triggered by data id {}.", data_id))
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: Similar as above, I would change this to `logger.debug(__("Manager processing communicate command triggered by Data with id {}.", data_id))`.

File: resolwe/flow/managers/dispatcher.py [hunk 9]
```diff
@@ -129,11 +129,15 @@ def __enter__(self):
```
Comment (tjanez) [inline resolwe/flow/managers/dispatcher.py]: I think this commit also deserves a mention in the CHANGELOG.

File: resolwe/flow/management/commands/list_docker_images.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (tjanez) [reply resolwe/flow/management/commands/list_docker_images.py]: Ok, I see. Thanks for the explanation!

File: tests/settings.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tjanez) [reply tests/settings.py]: I'm a bit confused. Did you use the `RotatingFileHandler` only to limit the size of the log file and not to actually rotate the logs between files?

**Pull request merged**